libs/components/lookup/testing/src/country-field/country-field-harness.ts

```diff
@@ -0,0 +1,174 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyOverlayHarness } from '@skyux/core/testing';
+
+import { SkyAutocompleteHarness } from '../autocomplete/autocomplete-harness';
+import { SkyAutocompleteInputHarness } from '../autocomplete/autocomplete-input-harness';
+
+import { SkyCountryFieldHarnessFilters } from './country-field-harness-filters';
+import { SkyCountryFieldSearchResultHarness } from './country-field-search-result-harness';
+import { SkyCountryFieldSearchResultHarnessFilters } from './country-field-search-result-harness-filters';
+
+/**
+ * Harness for interacting with a country field component in tests.
+ */
+export class SkyCountryFieldHarness extends SkyAutocompleteHarness {
+  /**
+   * Finds a standard country field component, or a country field component that is wrapped by an input box component.
+   * For input box implementations, we need to use the `.sky-input-box` selector since the `sky-country-field`
+   * element is removed from the DOM.
+   * @internal
+   */
+  public static override hostSelector = 'sky-country-field,.sky-input-box';
+
+  #documentRootLocator = this.documentRootLocatorFactory();
+
+  #getInput = this.locatorFor(SkyAutocompleteInputHarness);
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyCountryFieldHarness` that meets certain criteria.
+   */
+  public static override with(
+    filters: SkyCountryFieldHarnessFilters,
+  ): HarnessPredicate<SkyCountryFieldHarness> {
+    return SkyCountryFieldHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Blurs the country field input.
+   */
+  public override async blur(): Promise<void> {
+    return await super.blur();
+  }
+
+  /**
+   * Clears the country field input value.
+   */
+  public override async clear(): Promise<void> {
+    return await super.clear();
+  }
+
+  /**
+   * Enters text into the country field input.
+   */
+  public override async enterText(value: string): Promise<void> {
+    return await super.enterText(value);
+  }
+
+  /**
+   * Focuses the country field input.
+   */
+  public override async focus(): Promise<void> {
+    return await super.focus();
+  }
+
+  /**
+   * Gets the country field  `aria-labelledby` value.
+   * This is not needed for country field because the id is generated internally.
```

Comment:
can we be more descriptive that this is being marked `@internal` to remove this value from the JSdocs for country field?

Reply:
Updated